### DISCUSSION TITLE
Certain item-validation config options don't seem to work.

### DISCUSSION THREAD

**Discussion by Joko991** [PaperMC/Paper#10601] [Ideas]
### Expected behavior

Changing the maximum length of the page, and max number of bytes in a page to -1 should allow for the player to use these books without these limits.

### Observed/Actual behavior

The "Book too large!" kick seems to activate no matter what the config options are set to, page-max seems to work with all positive numbers up to 2147483647 so it is set to that, but the maximum length of page seems to be unchanged no matter what is in the config.

### Steps/models to reproduce

1.Have a paper server with default options.
2. Set "page" and "page-max" under item-validationto 2147483647, -1, or some other very large number.
3. If page-max is set to -1, books are disabled. No matter what "page" is set to the kick always occurs at around the same length.

### Plugin and Datapack List

Plugins:
Bukkit Plugins:
[22:13:17 INFO]:  
- 
CoreProtect
, 
floodgate
, 
Geyser-Spigot
, 
SkinsRestorer
, 
ViaVersion
Datapacks:
[22:13:22 INFO]: There are 2 data pack(s) enabled: 
[
vanilla (built-in)
]
, 
[
file/bukkit (world)
]
[22:13:22 INFO]: There are no more data packs available

### Paper version

[22:13:25 INFO]: 
Checking version, please wait...
[22:13:25 INFO]: 
This server is running Paper version git-Paper-461 (MC: 1.20.4) (Implementing API version 1.20.4-R0.1-SNAPSHOT) (Git: b600140)
You are running the latest version
Previous version: git-Paper-459 (MC: 1.20.4)

### Other

Page-max has been set to 2147483647 on my server, "page" has been tested on 2147483647, 16777216, default configuration, and -1, all of them provided the same result from what I saw.
total-multiplier is set to 1.0, so that should not be an issue.
The player is not an operator, and is on Java edition.

**Comment by electronicboy** [comment]
This is basically a feature request, not a bug report; setting a stupidly large number will also cause the limits to overflow, so, maaaybe that could be considered an oversight, but there was generally 0 expectation somebody would set the value that high

**Comment by Joko991** [comment]
Ah, I thought this would be considered a bug report. From what I saw in console the server doesn't start and gives you an error if any of the values are too high/low, so I thought the values I have currently would be fine. Would it be possible for if it is set to a negative number or just -1, for the check to be disabled?

**Comment by aHalliday13** [comment]
Has there been any updates on this? I run a small paper server, and it's infuriating to get kicked making books sometimes as small as six pages, and not being able to modify the configuration in any meaningful way to fix it. I agree with joko that there should be some way to just disable the check.

**Comment by Machine-Maker** [comment]
Should be fixed in https://github.com/PaperMC/Paper/commit/098bd39092f50cb97a604c2f228f2af29666ca41
